src/components/ui/ScrollArea/stories/ScrollArea.stories.tsx

```
@@ -44,3 +44,30 @@ export default {
 All.args = {};
+
+const LayoutTemplate = () => {
+    return <SandboxEditor>
+        <div className='w-full h-screen bg-gray-200 text-gray-950 border-2 border-gray-300 max-h-screen'>
+            <ScrollArea.Root>
+                <ScrollArea.Viewport>
+                    <div className='w-full bg-gray-200 text-gray-950 p-4'>
+                        <Heading>Scroll Area</Heading>
+                        {Array.from({ length: 100 }).map((_, index) => (
+                            <>
+                                <Heading as='h2'>Scroll Area</Heading>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)